tfs/src/main/java/hudson/plugins/tfs/model/BuildCommand.java

```diff
@@ -145,10 +145,12 @@ else if ("git.pullrequest.merged".equals(eventType)) {
         if (pp != null && requestPayload.containsKey(TeamBuildEndpoint.PARAMETER)) {

```

Comment:
Can you now remove the "//noinspection" line above? I am not sure, but it seems like you should be able to since you are using pp later on.

Reply:
Done.